hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/OmTableInsightTask.java

```diff
@@ -376,11 +376,36 @@ private void handleUpdateEvent(OMDBUpdateEvent<String, Object> event,
     String countKey = getTableCountKeyFromTable(tableName);
     String unReplicatedSizeKey = getUnReplicatedSizeKeyFromTable(tableName);
     String replicatedSizeKey = getReplicatedSizeKeyFromTable(tableName);
-    // In Update event the count for the table will not change. So we don't
+
+    if (event.getValue() != null) {
+      if (sizeRelatedTables.contains(tableName)) {
+        // Handle update for only size related tables
+        handleSizeRelatedTableUpdateEvent(event, countKey, unReplicatedSizeKey,
```

Comment:
the `handleUpdateEvent` is handling only the size related table update events currently? if the `event.getValue()` is null we purposely don't do anything?